arrays/c-or-cpp/evenodd.c

```diff
@@ -0,0 +1,40 @@
+#include <stdio.h>

```

Comment:
Please fix ident of file

Reply:
Rename file name from `evenodd` to `even-add-odd-number`

Reply:
thats even and odd not even and add

Reply:
Only left this one @MannyP31